test/support/gravity_mock.ex

```diff
@@ -36,4 +37,12 @@ defmodule GravityMock do
       }
     }
   end
+
+  def match_partners(_term, _token) do
```

Comment:
this is mainly to fix a warning we are getting about mock missing these methods.